src/editors/containers/ProblemEditor/components/EditProblemView/index.test.tsx

```diff
@@ -99,27 +86,19 @@ describe('EditProblemView', () => {
   });
 
   it('renders markdown editor with RawEditor', () => {
-    const modifiedInitialState = {
+    const modifiedInitialState: PartialEditorState = {
       app: {
-        analytics: {},
         lmsEndpointUrl: null,
-        returnUrl: '/return',
         isMarkdownEditorEnabledForCourse: true,
       },
       problem: {
-        problemType: 'standard',
+        problemType: null,
         isMarkdownEditorEnabled: true,
-        completeState: {
-          rawOLX: '<problem></problem>',
-          rawMarkdown: '## Problem',
-        },
+        rawOLX: '<problem></problem>',
+        rawMarkdown: '## Problem',
         isDirty: false,
       },
     };
-    initializeMocks({
-      initializeStore,
-      initialState: modifiedInitialState,
-    });
```

Comment:
Since `initialState` can now be passed in to `editorRender`, we don't need all these redundant calls to `initializeMocks` with the same `initialState`